### PR TITLE
Docs 3816 docs hub improve article relationships core

### PR DESCRIPTION
**Pull request by clairewrtr**
These changes only involve adding article descriptions, tags and tag lists to add Related Content at base of article.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/CORE/CORETutorials/SystemConfiguration/ConfiguringtheSystemEmail.md [hunk 1]
```diff
@@ -25,4 +28,6 @@ Go to **System > Email** and enter a **From Name** for system emails.
 Next, select a **Send Mail Method** and fill out the remaining fields (SMTP) or log in (GMail OAuth).
 
 Click **SEND TEST MAIL** to verify the configured email settings are working.
-If the test email fails, double-check that the root user**Email** field is correctly configured.
\ No newline at end of file
+If the test email fails, double-check that the root user**Email** field is correctly configured.
```
Comment (BonnieF) [inline content/CORE/CORETutorials/SystemConfiguration/ConfiguringtheSystemEmail.md:31]: There needs to be a space between User and **Email** on line 31

File: content/CORE/UIReference/Accounts/Users.md [hunk 2]
```diff
@@ -61,4 +67,6 @@ The **Users** screen lets you create and manage user accounts.
 | Lock User | Prevents user from logging in or using password-based services. |
 | Permit Sudo | Lets user use `sudo`. |
 | Microsoft Account | Allows Windows authentication methods. |
-| Samba Authentication | Lets user suthenticate to Samba shares. |
\ No newline at end of file
+| Samba Authentication | Lets user suthenticate to Samba shares. |
```
Comment (BonnieF) [inline content/CORE/UIReference/Accounts/Users.md:70]: Lets users authenticate to Samba shares.

File: content/CORE/UIReference/Storage/Pools/DatasetsScreen.md [hunk 3]
```diff
@@ -52,7 +55,7 @@ Use **ADVANCED OPTIONS** to add additional dataset settings such as quota manage
 
 Additional **Other Options** Settings
 
-| Setting | Description |
+| Name | Description |
 |---------|-------------|
 | **Read-only** | Select an option from the dropdown list. **On** prevents the dataset being modified. **Off** allows users accessing the dataset to modify its contents. |
 | **Exec** | Select an option from the dropdown list. **On** allows processes to executd from within this dataset. **Off** prevents processes from executing in the dataset. It is recommended to set to **On**. |
```
Comment (BonnieF) [inline content/CORE/UIReference/Storage/Pools/DatasetsScreen.md:61]: The line below this one
**Snapshot directory ** should be **Snapshot directory**

**Review by BonnieF — CHANGES_REQUESTED** [review]
There are a couple of minir changes needed but otherwise, it looks great

File: content/CORE/UIReference/Storage/Pools/DatasetsScreen.md (hunk 3 above)
Comment (micjohnson777) [reply content/CORE/UIReference/Storage/Pools/DatasetsScreen.md:61]: On the table headings, Claire had it correct with | Settings | Description |

File: content/CORE/GettingStarted/StoringData.md [hunk 4]
```diff
@@ -63,4 +66,6 @@ Organize the pool with additional datasets or zvols according to your access and
 
 When you're finished building and organizing your TrueNAS pools, move on to configuring how the system [shares data]({{< relref "SharingStorage.md" >}})
 
-{{< taglist tag="corestoredata" limit="10" >}}
\ No newline at end of file
+{{< taglist tag="corestoredata" limit="10" >}}
+
+{{< taglist tag="coregettingstarted" limit="10" title="Related Articles" >}}
```
Comment (micjohnson777) [inline content/CORE/GettingStarted/StoringData.md:71]: In this second taglist shortcode suggest you change the title to this:
```suggestion
{{< taglist tag="coregettingstarted" limit="10" title="Related Getting Started Articles" >}}
```

File: content/CORE/UIReference/JailsPluginsVMs/Plugins/PluginsScreens.md [hunk 5]
```diff
@@ -123,3 +127,5 @@ For information on Plugins and Jails screens and how to add or manage jails see:
 [Managing Jails]({{< relref "/CORE/CORETutorials/JailsPluginsVMs/Jails/ManagingJails.md" >}})
 
 [Setting Up Jail Storage]({{< relref "/CORE/CORETutorials/JailsPluginsVMs/Jails/SettingUpJailStorage.md" >}})
+
+{{< taglist tag="coreapps" limit="10" title="Related Plugins Articles" >}}
```
Comment (micjohnson777) [inline content/CORE/UIReference/JailsPluginsVMs/Plugins/PluginsScreens.md:131]: The taglist shortcode is a bit confusing. you used tag="coreapps" but have decided to use title="Related Plugins Articles", why not just use tag="coreplugins" instead? Also if using only one taglist shortcode you don't need to include the title="" parameter.

File: content/CORE/UIReference/Storage/Pools/DatasetsScreen.md (hunk 3 above)
Comment (clairewrtr) [reply content/CORE/UIReference/Storage/Pools/DatasetsScreen.md:61]: Table issue (Snapshot directory) has been fixed.
It was my understanding that the style for CORE tables would be "Name | Description" and SCALE would be "Setting  | Description" to further differentiate between them.

File: content/CORE/GettingStarted/StoringData.md (hunk 4 above)
Comment (clairewrtr) [reply content/CORE/GettingStarted/StoringData.md:71]: Completed.

File: content/CORE/UIReference/JailsPluginsVMs/Plugins/PluginsScreens.md (hunk 5 above)
Comment (clairewrtr) [reply content/CORE/UIReference/JailsPluginsVMs/Plugins/PluginsScreens.md:131]: The tag is now "coreplugins." The taglist on PluginsScreens.md no longer has the title parameter. The title parameter appears on GettingStarted\Applications.md which this article links to and which has more than one taglist shortcode.

File: content/CORE/UIReference/Storage/Pools/DatasetsScreen.md (hunk 3 above)
Comment (micjohnson777) [reply content/CORE/UIReference/Storage/Pools/DatasetsScreen.md:61]: Never mind Claire. Pay no attention to the now deleted comment.

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**